bahmni_api_feed/models/order_save_service.py

```diff
@@ -488,7 +488,8 @@ def _get_product_ids(self, order):
         if order['productId']:
             prod_ids = self.env['product.product'].sudo().search([('uuid', '=', order['productId'])])
             if not prod_ids:
-               raise UserError ("Product does not exits in ERP or Inactivated in ERP master.")
+                #log message indicating this product/service was skipped, because it wasn’t found in Odoo.
+                _logger.warning("Order Id [%s] unprocessed as [%s] %s does not exists or is inactive in master", order["orderId"], order["productId"], order['productName'])
```

Comment:
can we make it as `partially processed` instead of `unprocessed` @rahu1ramesh @hari1119 ??

Reply:
@mohan-13 The `orderId` would be unique for that particular product. As discussed, if a product is missing then that order (not the whole quotation) will be skipped. I feel that unprocessed makes more sense than partially processed.